npm/iptm/iptm.go

```diff
@@ -201,18 +221,26 @@ func (iptMgr *IptablesManager) checkAndAddForwardChain() error {
 		Specs: []string{
 			util.IptablesJumpFlag,
 			util.IptablesAzureChain,
+			util.IptablesModuleFlag,
```

Comment:
Are we doing ct new condition for when the flag is false too? We will need to test this scenario too. Through conformance and cyclonus

Reply:
we are. We'll make this change in v2 as well